### DISCUSSION TITLE
Global zip file comment support

### DISCUSSION THREAD

**Discussion by JCarlesVilaseca** [gildas-lormeau/zip.js#231] [Q&A] [answered]
Is there a way to add a global zip file comment? (Not a entry comment)

**Comment by gildas-lormeau** [comment, marked as answer]
Yes, you can pass it to the `ZipWriter#close` method.

**Reply by JCarlesVilaseca** [reply]
Oh! On close()! Thank's!

**Comment by gildas-lormeau** [comment]
Here's a simple test using it: https://github.com/gildas-lormeau/zip.js/blob/master/tests/pages/test-wrapped.js#L19
